client/main.py

```diff
@@ -1,9 +1,11 @@
 import argparse
 import json
+import uvicorn
+
```

Comment:
According to PEP8 style guide, you should split sodlib imports and 3d-party imports with only one blank line.

https://peps.python.org/pep-0008/#imports